packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -22,6 +23,14 @@ describe('Sayt Component', () => {
       expect(sayt).to.be.an.instanceof(LitElement);
     });
 
+    it('should call setDebounce()', () => {
+      const setDebounceStub = stub(Sayt.prototype, 'setDebounce');
```

Comment:
As a general comment for changes throughout the file, we have gone with the pattern of not including "Stub" in the name of stubbed functions.